crates/blockifier/oz_erc20/Scarb.lock

```diff
@@ -1,14 +0,0 @@
-# Code generated by scarb DO NOT EDIT.
```

Comment:
Why delete this? If the cargo lockfile goes in git, shouldn't this one?

Reply:
oh, maybe when I deleted it in order to debug, it deleted from git, and I forgot to return it back :D